source/blueprints/banking.rst

```diff
@@ -26,45 +26,41 @@ Here are some examples of scripted dialogue interactions for conversational flow
 
 3. Domain-Intent-Entity Hierarchy
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Here is the NLP model hierarchy for our food ordering application.
-
-.. image:: /images/food_ordering_hierarchy.png
-    :width: 700px
-    :align: center
-
-The single domain, ``ordering``, encompasses all of the functionality required to order food for delivery from nearby restaurants. The ``ordering`` domain supports the following intents:
+The banking blueprint is organized into five domains: ``Greeting``, ``Accounts & Credit Cards``, ``FAQ``, and ``Unsupported``. The main domain, ``accounts_creditcards``, encompasses all of the functionality required to perform tasks that require personal user banking information. The ``accounts_creditcards`` domain supports the following intents:
 
    - ``greet`` — User wants to start a conversation
-   - ``build_order`` — User wants to make selections for food delivery
-   - ``place_order`` — User wants to confirm and place an order
-   - ``start_over`` — User wants to abandon current selections and restart the ordering process
+   - ``transfer_balances`` — User wants to transfer money from one account to another
+   - ``check_balances`` — User wants the current balance for a particular account
+   - ``pay_creditcard`` — User wants to pay off a credit card bill
+   - ``check_due_date`` — User wants the date the bill is due 
+   - ``setup_autopay`` — User wants to activate automatic payments
    - ``exit`` — User wants to end the current conversation
    - ``help`` — User is confused or needs instructions on how to proceed
-   - ``unsupported`` — User is talking about something other than food ordering
+   - ``unsupported`` — User is talking about something other than banking tasks
+
+The full list of intents for all domains is illustrated below.
+
+.. image:: /images/hierarchy_home_assistant.png
 
-For this app, only the ``build_order`` intent requires entity recognition. This intent supports the following entity types:
+There are two types of entities in MindMeld: :ref:`System Entities <system-entities>` and :doc:`Custom Entities <../userguide/entity_recognizer>`. System entities are pre-defined in MindMeld. Examples include ``sys_amount-of-money``, ``sys_time``, and ``sys_interval``. Custom entities are defined by the developers of each application. Within each entity folder, the file ``gazetteer.txt`` contains the full list of values for each custom entity.
 
-   - ``restaurant`` — The name of a restaurant location
-   - ``cuisine`` — The name of a cuisine
-   - ``category`` — The name of a food category on a restaurant menu
-   - ``dish`` — The name of a dish on a restaurant menu
-   - ``option`` — The name of an available option (customization, add-on, etc.) for a dish
-   - ``sys_number`` — The quantity of a given dish, captured by the :doc:`number system entity <../userguide/entity_recognizer>`
+Banking blueprint defines and uses the following custom entities in the ``accounts_creditcards`` domain, which are shown below:
```

Comment:
Update to: The banking app defines and uses the following custom entities in the ``accounts_creditcards`` domain: